systemd/ics2entropiawiki.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Retrieve ics File and publish it to the entropia wiki
+
+[Service]
+Type=simple
+ExecStart=/usr/local/bin/ics2entropiawiki -c /etc/ics2entropiawiki/config.ini
+User=ics2entropiawiki
+Group=systemd-journal
```

Comment:
why?

Reply:
Bare with me here, I have never written a system-wide systemd service file.
What would you recommend?